### ISSUE TITLE
[Winerror 64] running zti, help needed

### ISSUE THREAD

**Issue by pjfarleyiii** [IBM/tnz#153]
I am trying to use `zti` to connect to the 3270 port on the public IBM Zxplore system.  My regular 3270 emulator (Vista TN3270) connects to this IP address and port with no problem.  I have attached a tnz.log file after changing the log level to DEBUG in a private `dbg-logging.json` file.

Steps to reproduce: In a Windows 10 directory which contains the `dbg-logging.json` file:
```
set TNZ_LOGGING=dbg-logging.json
set ZTI_SECLEVEL=0
zti 204.90.115.200:623
```
Any assistance or advice on how to resolve this error is appreciated.

[tnz.log](https://github.com/user-attachments/files/17286133/tnz.log)
[dbg-logging.json](https://github.com/user-attachments/files/17286134/dbg-logging.json)

**Comment by najohnsn** [comment]
I've never seen that sort of error before: `OSError: [WinError 64] The specified network name is no longer available
`. I was able to recreate though. It seems to have to do with the secure connection. I was able to connect to that host by turning off SSL. I _think_ that is what you intended to do with `ZTI_SECLEVEL`, but `ZTI_SECLEVEL=1` is the only supported setting:
```
set TNZ_LOGGING=dbg-logging.json
set SESSION_SSL=0
zti 204.90.115.200:623
```
